### ISSUE TITLE
All Interpretations - Continue Work via PR?

### ISSUE THREAD

**Issue by JordanMartinez** [chexxor/purescript-documentation-discussion#46]
Now that you've finished the last source, do you mind if we do any modifications to the `All Interpretations.md` file via PR? I've noticed some changes that you've made there and I'm not always sure what the reasoning behind those changes were. I'm also not sure whether I agree with such changes, so it'll be helpful to discuss such things.

However, I also realize that you are still limited in your time, so perhaps we shouldn't do that.

Either way, I think it's time (or at least almost time) for me to start writing the context/narrative part in a new PR, which you can examine and provide feedback on

Thoughts?

**Comment by chexxor** [comment]
Here's a PR to move that last file I wanted to move into that All Interpretations file: https://github.com/chexxor/purescript-documentation-discussion/pull/47

Yeah, IIRC, the last few changes I made to the All Interpretations file were motivated by my want to improve clarity in the ToC and to move a few items in that file from "uncategorized" to a pre-existing category. Yeah, we should consider that file to be relatively locked now because you will be building the context/narrative part based on it.

**Comment by JordanMartinez** [comment]
> Yeah, IIRC, the last few changes I made to the All Interpretations file were motivated by my want to improve clarity in the ToC and to move a few items in that file from "uncategorized" to a pre-existing category.

Ok. I think it surprised me a bit because it seemed like some headers were split into even smaller headers that did not seem to add much clarity / made minor points more emphasized...? I can't recall since it's been a while since I looked at it. 
However, that might just show that I was reading meaning into some headers and perhaps you did not read the same meaning into the headers due to how I worded it.

I do recall one interpretation showing up that I didn't think was correct. It read "One person writes documentation and finds users don't consult documentation even when they know they should. [Discussion_about_WNTYAD]"

However, looking back at the source, I no longer see the comments I recall reading that clarified what that person meant. I thought they were referring to documentation for people who use some application (e.g. end-user documentation on how to use Excel formulas), not docs that developers use to write applications (e.g. developer documentation on how to write Excel add-ins).
Regardless, I think the comment still holds some truth even for developer documentation. So, let's just leave it be.

> Yeah, we should consider that file to be relatively locked now because you will be building the context/narrative part based on it.

Ok, sounds good!
